feature-libs/organization/account-summary/components/services/account-summary-item.service.ts

```diff
@@ -0,0 +1,53 @@
+import { Injectable } from '@angular/core';
+import { B2BUnit, RoutingService } from '@spartacus/core';
+import {
+  CurrentUnitService,
+  ItemService,
+  UnitFormService,
+} from '@spartacus/organization/administration/components';
+import { OrganizationItemStatus } from '@spartacus/organization/administration/core';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AccountSummaryItemService extends ItemService<B2BUnit> {
```

Comment:
can we make `AccountSummaryItemService` not extend `ItemService`?

Reply:
```
export class AccountSummaryItemService {
  constructor(protected routingService: RoutingService) {}

  launchDetails(item: B2BUnit): void {
    const detailsPageRoute = this.getDetailsRoute();
    if (detailsPageRoute && item && Object.keys(item).length > 0) {
      this.routingService.go({
        cxRoute: detailsPageRoute,
        params: item,
      });
    }
  }

  protected getDetailsRoute(): string {
    return 'orgAccountSummaryDetails';
  }
}
```

Reply:
yes we can